segmentation/include/pcl/segmentation/grabcut_segmentation.h

```diff
@@ -281,13 +278,13 @@ namespace pcl
 
         private:
         /// sum of r,g, and b
-        Eigen::Vector3f sum_;
+        Eigen::Vector3f sum_{Eigen::Vector3f::Zero ()};
         /// matrix of products (i.e. r*r, r*g, r*b), some values are duplicated.
-        Eigen::Matrix3f accumulator_;
+        Eigen::Matrix3f accumulator_{Eigen::Matrix3f::Zero ()};
         /// count of color samples added to the gaussian
-        std::uint32_t count_;
+        std::uint32_t count_{0};
         /// small value to add to covariance matrix diagonal to avoid singular values
-        float epsilon_;
+        float epsilon_{0.001f};
```

Comment:
```suggestion
        float epsilon_;
```
The variable is initialized in the constructor. Initializing it also here is confusing (and the default value in the constructor is `0.0001f`, not `0.001f`)